pkg/operator/operator/k8s.go

```diff
@@ -339,6 +339,7 @@ func AsyncGatewayContainers(api spec.API, queueURL string) kcore.Container {
 			"-bucket", bucket,
 			"-cluster", clusterName,
 			"-port", s.Int32(DefaultPortInt32),
+			"-cluster-config-path", "/configs/cluster/cluster.yaml",
```

Comment:
the value should probably be a constant, since it's used in more than one place (here and in the env vars)